sklearn/feature_extraction/tests/test_text.py

```diff
@@ -832,6 +832,28 @@ def test_vectorizer_min_df():
     assert len(vect.stop_words_) == 5
 
 
+def test_vectorizer_max_df_unwanted_float():
```

Comment:
We could have a single test with parametrization. The parameter could be passed as a parameter of the function as well as the error message and error type that is expected.